config.h.in

```diff
@@ -9,6 +9,10 @@
 #define SAUNAFS_PACKAGE_VERSION_MICRO @PACKAGE_VERSION_MICRO@
 #define SAUNAFS_PACKAGE_VERSION "@PACKAGE_VERSION@"
 #define SAUNAFS_VERSHEX ((@PACKAGE_VERSION_MAJOR@)*0x10000+(@PACKAGE_VERSION_MINOR@)*0x100+(@PACKAGE_VERSION_MICRO@))
+#ifdef _WIN32
+#define WINDOWS_ClIENT_VERSION "@WINCLIENT_PACKAGE_VERSION@"
+#define WINFSP_LIBRARY_VERSION "@WINFSP_VERSION@"
```

Comment:
Looks like these macros are not used in this project. Please consider moving them to the windows submodule.